crates/zksync_error/Cargo.toml

```diff
@@ -9,14 +9,25 @@ license.workspace = true
 repository.workspace = true
 version.workspace = true
 
+[features]
+default = ["std", "use_anyhow", "use_serde"]
+std = [ "serde/std", "lazy_static/spin_no_std", "anyhow?/std", "strum/std"]
+use_anyhow = ["dep:anyhow"]
+use_serde = ["dep:serde"]
+runtime_documentation = ["dep:serde", "dep:serde_json"]
+serialized_errors = ["dep:serde", "dep:serde_json"]
+packed_errors = ["use_serde"]
+
+
 [dependencies]
-anyhow.workspace = true
+anyhow = { workspace = true, optional = true }
+lazy_static = { version = "1.5.0", default-features = false, optional = true }
+serde = { version = "1.0.210", features = [ "derive", "alloc" ], default-features = false, optional = true }
+serde_json = { version = "1.0.128", optional = true }
+strum = { version = "0.26.3", default-features = false, features = ["derive"] }
+strum_macros = { version = "0.26.4", default-features = false }
```

Comment:
Why aren't you using deps from workspace anymore (e.g. `serde_json.workspace = true`) ?